translations/credits.tex/ua.po

```diff
@@ -154,6 +154,9 @@ msgstr ""
 #: sections/credits.tex:25
 #, no-wrap
 msgid ""
+"\\phantom{Translators placeholder}\n"
+"\n"
+msgstr ""
 "\\textbf{Переклад}: Сергій Сергеєв\n"
```

Comment:
Don't shy away from your last name, and please say in what language you translated.